packages/styled/src/base.js

```diff
@@ -132,27 +132,31 @@ let createStyled: CreateStyled = (tag: any, options?: StyledOptions) => {
         newProps.ref = ref
 
         const ele = React.createElement(finalTag, newProps)
+        let possiblyStyleElement = <></>
```

Comment:
Do u happen to know if there is any potential perf difference between using a fragment like this and using a `Noop` component like in the referenced Next.js PR? The intuition tells me that if there is any then probably the fragment approach is better but it probably also doesn't matter

Reply:
I noticed this as well when going over the referenced PR again. Maybe Fragments are less robust if they're implemented differently in various JSX runtimes. A no-op (`() => null`) is probably safer.

But even if there's no difference, let's follow nextjs (first come, first serve) to establish a de-facto standard.